Bug report: Webpack build fails with Invalid generator object for overlapping asset/inline rules

**Describe the bug**
The webpack build fails with a `ValidationError` caused by two overlapping module rules for `\.(png|jpg|gif)$` files. When a file matches both rules simultaneously, webpack merges their configs — resulting in an `asset/inline` module that inherits `generator: { filename: ... }` from the `asset/resource` rule. The `filename` property is not valid for `asset/inline` modules, causing the build to fail.

**To Reproduce** Steps to reproduce the behaviour or a link to the recipe / input used to cause the bug:

1. Clone the CyberChef repository
2. Run `npm install`
3. Run the webpack build (e.g. `npm run build`)
4. Observe the following error:

```
Module not found: ValidationError: Invalid generator object. Asset Modules Plugin has been initialized using a generator object that does not match the API schema.
 - generator has an unknown property 'filename'. These properties are valid:
   object { binary?, dataUrl? }
   -> Generator options for asset/inline modules.
```

**Expected behaviour**
The build should complete successfully. First-party images (not in `node_modules`) should be emitted as resource files, and third-party images (in `node_modules`) should be inlined — with no rule overlap.

**Screenshots**
N/A — build-time error reproducible from the terminal.

**Root cause**
In `webpack.config.js`, two rules both match the same files:

```js
{ // First party images are saved as files to be cached
    test: /\.(png|jpg|gif)$/,
    exclude: /(node_modules|bmfonts)/,
    type: "asset/resource",
    generator: { filename: "images/[name][ext]" }
},
{ // Third party images are inlined
    test: /\.(png|jpg|gif)$/,
    exclude: /web\/static/,   // <-- too broad; does not restrict to node_modules
    type: "asset/inline",
},
```

Any image not under `node_modules`, `bmfonts`, or `web/static` (e.g. a first-party image in `web/images/`) matches both rules. Webpack merges all matching rules, so the final module ends up with `type: "asset/inline"` (last rule wins) but retains `generator: { filename: ... }` from the first rule — an invalid combination.

**Proposed fix**
Change the second rule's `exclude` to an `include` so it only matches node_modules images, making the two rules mutually exclusive:

```js
{ // Third party images are inlined
    test: /\.(png|jpg|gif)$/,
    include: /node_modules/,
    type: "asset/inline",
},
```

**Desktop (if relevant, please complete the following information):**
 - OS: Windows 11
 - Browser: N/A (build-time issue)
 - CyberChef version: current `master` branch

**Additional context**
This issue exists in the current `master` branch of the upstream repository. The rule comments correctly describe the intent ("First party" vs "Third party"), but the `exclude` on the second rule does not correctly implement it.

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -88,8 +88,8 @@ module.exports = {
                     from: "tesseract/**/*",
                     to: "assets/"
                 }, {
-                    context: "node_modules/tesseract.js/",
-                    from: "dist/worker.min.js",
+                    context: "node_modules/tesseract.js/dist",
+                    from: "worker.min.js",
                     to: "assets/tesseract"
                 }, {
                     context: "node_modules/tesseract.js-core/",
@@ -221,7 +221,7 @@ module.exports = {
             },
             { // Third party images are inlined
                 test: /\.(png|jpg|gif)$/,
-                exclude: /web\/static/,
+                include: /node_modules/,
                 type: "asset/inline",
             },
         ]

